<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_PowerElectrobot</class>
 <widget class="QMainWindow" name="ihm_PowerElectrobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_general">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="2">
           <widget class="QPushButton" name="push_all_outputs_off">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(204, 0, 0);</string>
            </property>
            <property name="text">
             <string>All Outputs OFF</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="current_out1">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>[A]</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-99.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Current Out2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="current_out2">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>[A]</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-99.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="global_current">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>[A]</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-99.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Current Out1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Battery Voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="battery_voltage">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>[V]</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Global Current</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" rowspan="4" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="output_1">
              <property name="text">
               <string>Output1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_2">
              <property name="text">
               <string>Output2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_3">
              <property name="text">
               <string>Output3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_4">
              <property name="text">
               <string>Output4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_5">
              <property name="text">
               <string>Output5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_6">
              <property name="text">
               <string>Output6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_7">
              <property name="text">
               <string>Output7</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_8">
              <property name="text">
               <string>Output8</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_calib">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Battery Voltage P1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="valid_calib_batt_voltag_p1">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Battery Voltage P2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="valid_calib_batt_voltag_p2">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Global Current P1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="valid_calib_global_current_p1">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Global Current P2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="valid_calib_global_current_p2">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Current Out1 P1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="valid_calib_current_out1_p1">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Current Out1 P2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="valid_calib_current_out1_p2">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Current Out2 P2</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="valid_calib_current_out2_p1">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Current Out2 P2</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="valid_calib_current_out2_p2">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="calib_batt_voltage_p1">
          <property name="suffix">
           <string>[mV]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSpinBox" name="calib_current_out2_p2">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="calib_current_out2_p1">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="calib_current_out1_p2">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="calib_current_out1_p1">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="calib_global_current_p2">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="calib_global_current_p1">
          <property name="suffix">
           <string>[mA]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="calib_batt_voltage_p2">
          <property name="suffix">
           <string>[mV]</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_special">
       <attribute name="title">
        <string>Special</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="valid_reset_eeprom_factory">
          <property name="text">
           <string>Reset Factory EEPROM</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Change @I2C</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="new_i2c_address">
          <property name="toolTip">
           <string>New I2C address</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string>0x</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="valid_i2c_address">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>213</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="valid_unlock_eeprom">
          <property name="text">
           <string>Unlock EEPROM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
